medusa/storage/abstract_storage.py

```diff
@@ -261,3 +261,9 @@ def additional_upload_headers(self):
         child classes.
         """
         return {}
+
+    def get_storage_class(self):
+        if self.config.storage_class is not None:
+            return self.config.storage_class
+        else:
+            return None
```

Comment:
```suggestion
        return self.config.storage_class
```

I guess we can simplify this method since this should be equivalent.